os/net/mac/csma/csma.c

```diff
@@ -72,6 +72,8 @@ input_packet(void)
                                          &linkaddr_node_addr) &&
             !packetbuf_holds_broadcast()) {
     LOG_WARN("not for us\n");
+  } else if(linkaddr_cmp(packetbuf_addr(PACKETBUF_ADDR_SENDER), &linkaddr_node_addr)) {
+    LOG_WARN("frame from ourselves\n");
```

Comment:
should this be done here or upper in the stack? If done at MAC we must make sure all MACs also do it